draft-ietf-dots-signal-channel-26.xml

```diff
@@ -925,17 +925,17 @@
 
           <t>FQDN and URI mitigation scopes may be thought of as a form of
           scope alias, in which the addresses associated with the domain name
-          or URI represent the full scope of the mitigation.</t>
+          or URI (as resolved by the server) represent the full scope of the mitigation.</t>
 
           <t>In the PUT request at least one of the attributes
           'target-prefix', 'target-fqdn','target-uri', or 'alias-name' MUST be
           present.</t>
 
           <t>Attributes and Uri-Path parameters with empty values MUST NOT be
-          present in a request.</t>
+          present in a request and render the entire request invalid.</t>
```

Comment:
This is only borderline editorial, but I could not find an alternate interpretation that made sense.